example/react/src/CasbinDemo.js

```diff
@@ -0,0 +1,32 @@
+import React, {Component} from 'react';
+import {Authorizer} from 'casbin.js';
+import permissions from './permissions.json'
+
+export default class CasbinDemo extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      user: props.user
+    }
+    this.auth = new Authorizer("manual");
+  }
+
+  componentWillReceiveProps(nextProps) {
+    this.setState({user: nextProps.user});
+  }
+
+  render() {
+    // set the permission when the user identity changes.
+    this.auth.setPermission(permissions[this.state.user]);
+    const alice_div = <p>This is alice's data.</p>
```

Comment:
please use Camel-Case: `const AliceDiv = <p>This is alice's data.</p>`

Reply:
I think `aliceDiv` is better?